src/pages/workspace/accounting/intacct/export/SageIntacctNonReimbursableExpensesPage.tsx

```diff
@@ -0,0 +1,166 @@
+        hasError:
+            config?.export.nonReimbursable === CONST.SAGE_INTACCT_NON_REIMBURSABLE_EXPENSE_TYPE.VENDOR_BILL
+                ? !!config?.export?.errorFields?.nonReimbursableVendor
+                : !!config?.export?.errorFields?.nonReimbursableCreditCardChargeDefaultVendor,
+        pendingAction:
+            config?.export.nonReimbursable === CONST.SAGE_INTACCT_NON_REIMBURSABLE_EXPENSE_TYPE.VENDOR_BILL
+                ? config?.export?.pendingFields?.nonReimbursableVendor
+                : config?.export?.pendingFields?.nonReimbursableCreditCardChargeDefaultVendor,
+    };
+
+    const defaultVendor = (
+        <OfflineWithFeedback
+            key={defaultVendorSection.description}
+            pendingAction={defaultVendorSection.pendingAction}
+        >
+            <MenuItemWithTopDescription
+                title={defaultVendorSection.title}
+                description={defaultVendorSection.description}
+                shouldShowRightIcon
+                onPress={defaultVendorSection.action}
+                brickRoadIndicator={defaultVendorSection.hasError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+            />
+        </OfflineWithFeedback>
+    );
+
+    const creditCardAccountSection = {
+        description: translate('workspace.sageIntacct.creditCardAccount'),
+        action: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_NON_REIMBURSABLE_CREDIT_CARD_ACCOUNT.getRoute(policyID)),
+        title: config?.export.nonReimbursableAccount ? config.export.nonReimbursableAccount : translate('workspace.sageIntacct.notConfigured'),
+        hasError: !!config?.export?.errorFields?.nonReimbursableAccount,
+        pendingAction: config?.export?.pendingFields?.nonReimbursableAccount,
+    };
+
+    const creditCardAccount = (
+        <OfflineWithFeedback
+            key={creditCardAccountSection.description}
+            pendingAction={creditCardAccountSection.pendingAction}
+        >
+            <MenuItemWithTopDescription
+                title={creditCardAccountSection.title}
+                description={creditCardAccountSection.description}
+                shouldShowRightIcon
+                onPress={creditCardAccountSection.action}
+                brickRoadIndicator={creditCardAccountSection.hasError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+            />
+        </OfflineWithFeedback>
+    );
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            testID={SageIntacctNonReimbursableExpensesPage.displayName}
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.sageIntacct.nonReimbursableExpenses.label')}
+                onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_EXPORT.getRoute(policyID))}
```

Comment:
We probably need to use ConnectionLayout here. Because we still need to block user from accessing this page if they don't have permission